libstorage-ng.spec.in

```diff
@@ -22,7 +22,7 @@ Version:        @VERSION@
 Release:        0
 Summary:        Library for storage management
 License:        GPL-2.0
-Group:          System/Libraries
+Group:          Development/Libraries/C and C++
```

Comment:
Why this change? What are the definitions for those two groups?

Reply:
System/Libraries: "Packages in this group should already be installed automatically because of a dependency." libstorage-ng.src.rpm (that is what line 25 touches here) however neither contains a .so file, nor does it get autoinstalled.
More group descriptions are on https://en.opensuse.org/openSUSE:Package_group_guidelines#System .